components/datetime/src/provider/calendar/skeletons.rs

```diff
@@ -36,7 +36,14 @@ impl icu_provider::DataMarker for DateSkeletonPatternsV1Marker {
 }
 #[cfg(feature = "experimental")]
 impl icu_provider::KeyedDataMarker for DateSkeletonPatternsV1Marker {
-    const KEY: icu_provider::DataKey = icu_provider::data_key!("datetime/skeletons@1[u-ca]");
+    const KEY: icu_provider::DataKey = icu_provider::data_key!(
+        "datetime/skeletons@1",
+        DataKeyMetadata::construct_internal(
```

Comment:
Is it really `construct_internal` if this is using it?

Reply:
It's an expansion of the `data_struct` macro and it's not code anyone should normally be writing except that I want this only compiled in experimental mode. The normal way to do this is to use `Default::default()` and modify the fields, but that's not `const`